app/src/main/java/com/p2p/presentation/truco/TrucoActivity.kt

```diff
@@ -53,4 +55,9 @@ class TrucoActivity : GameActivity<TrucoSpecificGameEvent, TrucoViewModel>() {
         }
     }
 
+    // unused.
+    override fun goToPlay() {
+        Unit
+    }
```

Comment:
Then we should delete this right?

Reply:
no se puede, hay que sobreescribirlo sí o sí porque es abstract